eslint.config.mjs

```diff
@@ -234,13 +233,13 @@ export default tseslint.config(
         },
 
         plugins: {
-            jest,
+            vitest,
```

Comment:
We also add this plugin for all files on line 39. Can you delete that earlier declaration?

I also noticed that we don't have `__mocks__` anywhere, or any `__tests__` in integration-karma. Can you also clean up lines 226 and 247? Thanks!

Reply:
Nice catch! I cleaned up a bit.